app/api/routes/predict.py

```
@@ -1418,6 +1438,21 @@ async def predict_demo(
+        try:
+            demo_explanation = await generate_bet_explanation(
+                match.home_team,
+                match.away_team,
+                match.league,
+                best_bet.get("best_side", "home"),
+                float(best_bet.get("edge", 0)),
+                float(best_bet.get("odds", 1.85)),
+                consensus_prob,
+                confidence_val
+            )
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)